TeamCode/src/main/java/com/acmerobotics/relicrecovery/vision/OverlayView.java

```diff
@@ -0,0 +1,64 @@
+package com.acmerobotics.relicrecovery.vision;
+
+import android.content.Context;
+import android.graphics.Canvas;
+import android.view.View;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by ryanbrott on 9/23/17.
+ */
+public class OverlayView extends View {
```

Comment:
Is there functionality to get a camera and add trackers to it without having the display on the rc phone? I feel like that may impact preformance?